installer_linux/make_deb.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# preflight check
+
+if [[ ! -f ./make_deb.sh ]]; then
+	echo "You must run this script from within the installer_linux directory!"
+	exit 1
+fi
+
+# version
+
+if [ "$SURGE_VERSION" != "" ]; then
+    VERSION="$SURGE_VERSION"
+elif [ "$1" != "" ]; then
+    VERSION="$1"
+fi
+
+if [ "$VERSION" == "" ]; then
+    echo "You must specify the version you are packaging!"
+    echo "eg: ./make_deb.sh 1.0.6b4"
+    exit 1
+fi
+
+# Valid version for DEB start with a digit.
+DEB_VERSION=$VERSION
+if [[ ${DEB_VERSION:0:1} =~ [0-9] ]]; then
+    echo "DEB VERSION same as Version"
+else
+    DEB_VERSION="0.${VERSION}"
+    echo "DEB VERSION is ${DEB_VERSION}"
+fi
+
+
+# Names
+SURGE_NAME=Surge
+PACKAGE_NAME="$SURGE_NAME"
+# SURGE_NAME=surge-synthesizer
+
+# locations
+
+mkdir -p ${PACKAGE_NAME}/usr/lib/vst
+mkdir -p ${PACKAGE_NAME}/usr/lib/vst3
+mkdir -p ${PACKAGE_NAME}/usr/share/${SURGE_NAME}/doc
+mkdir -p ${PACKAGE_NAME}/DEBIAN
+
+# build control file
+
+if [[ -f ${PACKAGE_NAME}/DEBIAN/control ]]; then
+	rm ${PACKAGE_NAME}/DEBIAN/control
+fi
+touch ${PACKAGE_NAME}/DEBIAN/control
+cat <<EOT >> ${PACKAGE_NAME}/DEBIAN/control
+Package: ${PACKAGE_NAME}
+Version: $DEB_VERSION
+Architecture: amd64
+Maintainer: surgeteam
+Depends:libcairo2, libxkbcommon-x11-0, libxcb-util1, libxcb-cursor0
+Provides: vst-plugin
+Section: sound
+Priority: optional
+Description: Surge Synthesizer plugin
+EOT
+
+DATE=`date`
+MSG=`git log -n 1 --pretty="%s (git hash %H)"`
+cat <<EOT > ${PACKAGE_NAME}/DEBIAN/changelog
+${PACKAGE_NAME} (${DEB_VERSION}) stable; urgency=medium
+
+  * ${MSG}
+  * For more details see https://github.com/surge-synthesizer/surge
+
+ -- Surge Synthesizer Team <noreply@github.com>  ${DATE}
+EOT
+
+#copy data and vst plugins
+
+cp ../LICENSE ${PACKAGE_NAME}/usr/share/${SURGE_NAME}/doc
+cp -r ../resources/data/* ${PACKAGE_NAME}/usr/share/${SURGE_NAME}/
+cp ../target/vst2/Release/Surge.so ${PACKAGE_NAME}/usr/lib/vst/${SURGE_NAME}.so
+
+# Once VST3 works, this will be ../products/vst3
+# cp ../target/vst3/Release/Surge.so ${PACKAGE_NAME}/usr/lib/vst3/${SURGE_NAME}.so
+
+#build package
+
+dpkg-deb --build ${PACKAGE_NAME}
```

Comment:
`dpkg-deb --build ${PACKAGE_NAME} product/${PACKAGE_NAME}-linux-x64-${VERSION}.deb` ?

Reply:
Oh I guess yeah I can do that! Let me test and push